_posts/2019-02-07-graphql-and-google-calendar-api.markdown

```diff
@@ -3,6 +3,8 @@ layout: post
 title: "GraphQL and Google Calendar API"
 date: 2019-02-07 09:30
 categories: upcoming
```

Comment:
Could you change `categories: upcoming` to `categories: meetings`, so the post is moved to the past meetings group? 
Thanks!